src/orquestra/sdk/dremio/_api.py

```diff
@@ -0,0 +1,86 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+import certifi
+
+from .._base import _env
+from ._env_var_reader import EnvVarReader
+from ._flight_facade import FlightCallOptions, FlightClient, FlightDescriptor
+
+
+class DremioClient:
+    """Reads dataframes from Dremio instance running on Orquestra.
+
+    Requires specifying connection details as environment variables:
+
+        * ``ORQ_DREMIO_HOST``: the full dremio URI, with protocol and port. Example
+          value: ``grpc+tls://ws2-12de2f.test-cluster.dremio-client.orquestra.io``
+        * ``ORQ_DREMIO_USER``: your Dremio account email.
+        * ``ORQ_DREMIO_PASS``: your Dremio account password.
+
+    To obtain the values described above, follow these steps:
+
+    #. Open Orquestra Portal—open your cluster's URI in a web browser.
+    #. Select workspace.
+    #. Open "Connectors".
+    #. Click "Connect" on the "Dremio" tab.
+    #. Click "Copy Flight Endpoint". This is the value for your ``ORQ_DREMIO_URI``.
+    #. Click "Lanuch".
+    #. Inside Dremio, go to settings and configure your user account.
+       ``ORQ_DREMIO_USER`` is your Dremio account email.
+       ``ORQ_DREMIO_PASS`` is your Dremio account password.
+    """
+
+    @classmethod
+    def from_env_vars(cls) -> "DremioClient":
+        cert_contents = read_certificate()
+
+        uri_reader = EnvVarReader(_env.ORQ_DREMIO_URI)
+        user_reader = EnvVarReader(_env.ORQ_DREMIO_USER)
+        pass_reader = EnvVarReader(_env.ORQ_DREMIO_PASS)
+
+        flight_client = FlightClient(
+            f"{uri_reader.read()}",
+            tls_root_certs=cert_contents,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏weird that dremio doesn't get the system root certs by default

Reply:
Yeah, it's weird. I'll ask around the original authors.

Reply:
Turns out it's not needed for how we're using Flight API and `pyarrow>=10.0`